src/base/bittorrent/session.cpp

```diff
@@ -388,6 +388,7 @@ Session::Session(QObject *parent)
     , m_IDNSupportEnabled(BITTORRENT_SESSION_KEY("IDNSupportEnabled"), false)
     , m_multiConnectionsPerIpEnabled(BITTORRENT_SESSION_KEY("MultiConnectionsPerIp"), false)
     , m_validateHTTPSTrackerCertificate(BITTORRENT_SESSION_KEY("ValidateHTTPSTrackerCertificate"), true)
+    , m_mitigateSsrf(BITTORRENT_SESSION_KEY("MitigateSsrf"), true)
```

Comment:
```suggestion
    , m_mitigateSsrf(BITTORRENT_SESSION_KEY("SSRFMitigation"), true)
```

Reply:
@Chocobo1 
Either `MitigateSSRF` or `SSRFMitigationEnabled`. The former follows the similar setting name: `ValidateHTTPSTrackerCertificate`.

Reply:
Then `SSRFMitigationEnabled`.
"Mitigate" SSRF (a form of exploit) just doesn't sound right to me.